antlr4/PostgreSQLParser.g4

```diff
@@ -1316,6 +1333,17 @@ for_clause
     : FOR ( UPDATE | NO KEY UPDATE | SHARE | KEY SHARE ) (OF table_name_ (COMMA table_name_)*)? ( NOWAIT | SKIP_ LOCKED)*
     ;
 
+updater_clause
+    : (expr EQUAL expr (COMMA expr EQUAL expr)*)
+    | (OPEN_PAREN name_list CLOSE_PAREN EQUAL expr_list)
+    | (OPEN_PAREN name_list CLOSE_PAREN EQUAL OPEN_PAREN select_stmt CLOSE_PAREN)
+    ;
+
+returning_clause
+    : RETURNING ((column_name_=expr (AS? name_)?) | STAR)
```

Comment:
nit: assign a label to `name_` -> `output_name=name_`. We may not do a great job at this across the grammar, but I was planning on doing some normalization once I'm happy with the correctness of the grammar as defined by the success rates of the tests.

It's generally hard to get things consistent at this stage of the development because there are so many rules.

Reply:
Good idea. 
Done!